<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlastPit Setup</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QComboBox" name="profile_list">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>default</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="open_profile">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_profile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_profile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lr2path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LR2 Path</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lr2path">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_lr2path">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_folder">
    <attribute name="title">
     <string>Folder</string>
    </attribute>
    <widget class="QPushButton" name="add_folder">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="open_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QListWidget" name="folder_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>192</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_video">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QCheckBox" name="vsync">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VSync</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_resolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
    <widget class="QComboBox" name="resolution_list">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>640x480</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1920x1080</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_videomode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="windowed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Windowed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fullscreen">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FullScreen</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="borderless">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Borderless</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxfps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MaxFPS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxfps">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxfps_hint">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(0: disabled)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_audio">
    <attribute name="title">
     <string>Audio</string>
    </attribute>
    <widget class="QLabel" name="label_audio_backend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Backend</string>
     </property>
    </widget>
    <widget class="QComboBox" name="audio_backend_list">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_audio_buffer_count">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer Count</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="audio_buffer_count">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_audio_buffer_size">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="audio_buffer_size">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_audio_backend">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="asio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Check ASIO Devices</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_advanced">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QCheckBox" name="mouse_analog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse Analog</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_select_scroll_speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Song select scroll speed (ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="select_scroll_speed_1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_miss_bga_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Miss BGA time (ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="select_scroll_speed_2">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="miss_bga_time">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_min_input_interval">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min input interval (ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_input_interval">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_new_song_duration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New song duration (hour)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_song_duration">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="relative_axis">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Relative Axis (useful for controllers with turntable)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rungame">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>label_version</zorder>
  <zorder>rungame</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
